test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -286,4 +286,19 @@ def test_successful_validate_pinless_debit_card
     assert_equal 'Y', response.params["status"]
     assert_equal true,  response.success?
   end
+
+  def test_network_tokenization_authorize_and_capture
```

Comment:
> whether we should handle this or let the gateway handle it

Mmmm, yes, I do suppose the general approach of AM is to not introduce too much between the caller and the gateway. Ok!